e2e/e2etests/test_bitcoin_deposit_and_withdraw_with_dust.go

```diff
@@ -12,17 +13,28 @@ import (
 	crosschaintypes "github.com/zeta-chain/node/x/crosschain/types"
 )
 
+// wgDeposit is a wait group for deposit runner to finish
+var wgDepositRunner sync.WaitGroup
+
+func init() {
```

Comment:
Let's not use init in e2e package why do we need this waitgroup?

Reply:
The `TestBitcoinWithdrawRBFName` needs Bitcoin block generation to stop, so that the broadcasted tx will be pending in mempool. Otherwise, tx will be included in block right away and there is no chance to replace it.